<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementEditor</class>
 <widget class="QWidget" name="ElementEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="chkNillable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>74</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nillable</string>
   </property>
   <property name="tristate">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAbstract">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>74</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Abstract</string>
   </property>
   <property name="tristate">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Occurs</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Occurs</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minOccurs">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>49</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxOccurs">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>49</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default Value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="defaultValue">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fixed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Final</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbFinal">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBlock">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>type of object:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbElementCategory">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>references:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbReference">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
